docs/keepers.md

```diff
@@ -0,0 +1,51 @@
+# Keepers
+
+## Overview
+
+## Common Conventions
+
+The Keeper class follows similar conventions as the smart contracts. See [smart_contracts.md](./smart_contracts.md) for explanations.
+
+## Classes
+### Keeper
+
+The base interface for all other keeper classes. `generate_actions_for_timestep` returns an array of `action` objects that is run by [experiment.py](../experiments/experiment.py)
```

Comment:
would be helpful to explain the anatomy of an `action` object here

Reply:
well, either here or in `experiment.md`, worth thinking about

Reply:
I'll refer to it in `experiment.md`